src/components/styles/Button.styled.ts

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components'
+
+export const BaseButton = styled.div`
+  padding: 0.5rem 1rem;
+  .ant-btn-text {
+    height: auto;
+    font-size: 20px;
```

Comment:
Being a general purpose button, I feel that font size should be an optional prop and the size of the button should adjust accordingly

Reply:
Can leave default as 20px

Reply:
44248d0937ee3bc2c33bb07afdaa22e20c9677e9